bundles/org.openhab.ui.basic/web-src/_layout.scss

```diff
@@ -212,19 +212,20 @@
 		}
 	}
 	&__label {
+		padding-left: $form-row-desktop-padding;
+		html.ui-layout-condensed & {
+			padding-left: $form-row-desktop-padding-condensed;
+		}
 		@media screen and (max-width: $layout-tablet-size-threshold) {
 			padding-left: $form-row-mobile-padding;
 			html.ui-layout-condensed & {
 				padding-left: $form-row-mobile-padding-condensed;
 			}
 		}
-		padding-left: $form-row-desktop-padding;
-		html.ui-layout-condensed & {
-			padding-left: $form-row-desktop-padding-condensed;
-		}
 		@include flex-shrink(0);
 		@include flex-grow(2);
 		@include flex-2011(2 2 auto);
+		min-width: 5em;
```

Comment:
What is the impact of this added line on all other sitemap elements ?

Reply:
tbh, I'm not sure. From my limited testing with my own sitemaps, nothing looked weird.

Ideally we need to have some form of "unit test", e.g. a standard sitemap which has a rigorous set of combinations of elements so that when we try to render it, we could determine, at a glance, that everything "still looks ok" / not broken. That would have greatly aided in answering questions such as this.

Reply:
Do you remember what is the default value if not set ?
PS: I can check that.

Reply:
I believe it would've been 0 if not set specifically

Reply:
We have to see what happens with a small label for other widgets having a big control block. I tried with a slider with value displayed at left and that already looks good.

Reply:
This line has now been removed.